src/App.jsx

```diff
@@ -1,28 +1,59 @@
 import "./App.css";
 import GroupsPage from "./pages/GroupsPage";
+import Group from "./pages/Group";
 import ExpensesPage from "./pages/ExpensesPage";
 import { useState } from "react";
 
 function App() {
-  const [page, setPage] = useState('GroupsPage')
-  const [params, setParams] = useState('');
-  const props = {
-    page, setPage, params, setParams
-  }
-
-  let loadComponent;
-
-  if(page === 'GroupsPage') {
-    loadComponent = <GroupsPage {...props}/>
-  } else if(page === 'ExpensesPage') {
-    loadComponent = <ExpensesPage {...props}/>
-  }
-
-  return (
-    <div className="App">
-      {loadComponent}
-    </div>
-  );
+  // const [page, setPage] = useState('GroupsPage')
+  // const [params, setParams] = useState('');
+  // const props = {
+  //   page, setPage, params, setParams
+  // }
+
+  // let loadComponent;
+
+  // if(page === 'GroupsPage') {
+  //   loadComponent = <GroupsPage {...props}/>
+  // } else if(page === 'ExpensesPage') {
+  //   loadComponent = <ExpensesPage {...props}/>
+  // }
+
+  // return (
+  //   <div className="App gradiant_background">
+  //     {loadComponent}
+  //   </div>
+  // );
+
+  const debts = [
+    {
+      "personName": "Riya",
+      "amount": "100",
+      'currency': "$"
```

Comment:
- favour currency to take in `USD` so that localisation specific NumberFormatter can be used?